cmd/src/server.rs

```diff
@@ -476,10 +479,40 @@ impl TiKVServer {
 
         let import_path = self.store_path.join("import");
         let importer = Arc::new(SSTImporter::new(import_path).unwrap());
+
+        let mut split_check_worker = Worker::new("split-check");
+        let split_check_runner = SplitCheckRunner::new(
+            engines.engines.kv.clone(),
+            self.router.clone(),
+            coprocessor_host.clone(),
+            self.config.coprocessor.clone(),
+        );
+        split_check_worker.start(split_check_runner).unwrap();
+        cfg_controller.register(
+            "coprocessor",
+            Box::new(SplitCheckConfigManager(split_check_worker.scheduler())),
+        );
```

Comment:
Instead of using string of `TiKvConfig`'s filed name to register config manger, use `config::Module` maybe 
more clear and semantic, what do you think?